tests/Mock/DefaultAggregateRoot.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/*
+ * This file is part of the prooph/event-store.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 8/22/15 - 10:58 PM
+ */
+namespace Prooph\EventStoreTest\Mock;
+
+use Prooph\Common\Messaging\Message;
+
+/**
+ * Class DefaultAggregateRoot
+ *
+ * @package Prooph\EventStoreTest\Mock
+ * @author Alexander Miertsch <alexander.miertsch.extern@sixt.com>
+ */
+final class DefaultAggregateRoot implements DefaultAggregateRootContract
+{
+    private $historyEvents = [];
+
+    /**
+     * @param Message[] $historyEvents
+     * @return DefaultAggregateRootContract
+     */
+    public static function reconstituteFromHistory($historyEvents)
+    {
+        $self = new self();
```

Comment:
Perhaps a private constructor is nicer, but that's only a test mock, so I am okay with this.